### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by Shewitttt** [lab-brussels-1/hyf-redesign-group-1#2] [General]
Adding a discussion section for later.

**Comment by sammou00** [comment]
Hello, 
- `user-personas` : You have improved a lot , very good job.
- `Backlog`  :  try to keep the same style 
![image](https://user-images.githubusercontent.com/67711948/163688957-df19b566-70c3-432f-a6c8-3cd49aa64a14.png)
 As a user if they are more than one user personas just used as a user 
`As a user I want to see the website title ` 
![image](https://user-images.githubusercontent.com/67711948/163689007-40c9cae0-ca11-4aa4-b699-d3b7c767707c.png)
same style as a sponsor I want to know the values, missions and history of HYF

- I think the rest of planning is well enough good job.

**Comment by sammou00** [comment]


**Reply by sammou00** [reply]
your CSS has no comments at all, using a short and to the point comments is a very good idea.
![image](https://user-images.githubusercontent.com/67711948/163689295-eef03f6d-3c86-4a72-8a5f-623b54d693da.png)

**Comment by Shewitttt** [comment]
Thank you so much for your comments @samirm00. We will improve it accordingly :)

**Reply by sammou00** [reply]
Very good job. You have improved a lot.
